dictionary-octopus.txt

```diff
@@ -1,5 +1,6 @@
 ALLUSERSPROFILE
 apikey
+YOURAPIKEY
```

Comment:
This is the sample api key used in the page that I changed, it is used in a few other places, but there are a number of other examples of placeholder api keys used

Reply:
We're trying to standardize on the also-used `YOUR-API-KEY` which is a little more readable.

Reply:
I agree that it is more readable but it suggests a different format. Api Keys are always `API-XXXXX` with no additional hyphens.